ci/vale/dictionary.txt

```diff
@@ -0,0 +1,1474 @@
+citad
+cjones
+clearspace
+cli
+clickboard
+clickjacking
+client1
+client2
+clojure
+clonefrom
+clozure
+cls
+cluebringer
+cluster1
+clustermgr
+clustermgrguest
+cmd
+cmdmod
+cmusieve
+cnf
+codebase
+codeblock
+codec
+com2sec
+comodo
+conda
+config
+configs
+configtest
+confluece
+convolutional
+coose
+copyfrom
+copyto
+coreutils
+corporateclean
+couldn
+cpanminus
+cpu
+cqlsh
+cqlshrc
+craftbukkit
+craigslist
+crashingdaily
+createdt
+createfromstackscript
+cron
+crond
+cronie
+cronjob
+cronjobs
+crontab
+crt
+crypto
+csr
+css
+csum
+csv
+ctools
+ctrl
+customizability
+customizable
+cyberduck
+cyg
+cygwin
+cz
+d39a
+d693
+daemonization
+daemonize
+daemonized
+daemonizes
+daemonizing
+dahdi
+datadir
+dataset
+datasets
+datasource
+datastax
+datastore
+datbase
+datetime
+dav
+davfs
+davfs2
+db8
+dbconfig
+dbd
+dbdir
+dbfiles
+dbname
+dbserver
+dcs
+deathmatch
+debconf
+declaratively
+deduplication
+deeplearning
+defragment
+deprovisioning
+descolada
+descr
+dest
+dev
+devel
+devops
+devtmpfs
+df
+dhclient
+dialogs
+diavel
+didn
+diffie
+digium
+diplays
+dircaps
+directadmin
+dired
+displayfeatures
+disqus
+distro
+distros
+dists
+django
+dkim
+dmg
+dmitriy
+dmz
+dnf
+dns
+dnsleaktest
+dnsmasq
+dockercompose
+dockerfile
+dockerfiles
+dockerize
+dockerized
+dokku
+domain1
+domain2
+domain3
+domainkey
+domainname
+dotdeb
+dotfiles
+doveadm
+dpkg
+dport
+dropdown
+drupal
+drush
+drwxr
+dshield
+dsn
+ducati
+dust2
+dust2
+dvcs
+e2fsck
+ea04
+ead
+ebjabberd
+ecommerce
+ecute
+edu
+efi
+ejabberd
+el6
+el7
+elasticsearch
+elgg
+elpaso
+email2email
+ender
+enix
+entrypoint
+env
+envs
+epel
+erb
+erlang
+esc
+essage
+estart
+etcd
+etcd0
+eth0
+everytime
+evhost
+example1
+example2
+examplegroup
+examplepassword
+examplerole
+examplesitename
+exampleuser
+exe
+executables
+exim
+expirations
+ext2
+ext3
+ext4
+ext4
+facter
+facto
+fail2ban
+fail2ban
+failover
+failregex
+failregexs
+failsafes
+falko
+fastcgi
+fastmail
+fastopen
+fauxhai
+fb4c
+fcgi
+fcgiwrap
+fcrontab
+fd86
+fdisk
+fdriver
+fe70
+fedf
+fetchmail
+fetchmailrc
+ff02
+fi
+fiiiile
+file1
+filebeat
+filecap
+filecaps
+filemode
+filepath
+filepaths
+fileserver
+filesystem
+filesystems
+filezilla
+filimonov
+finnix
+firewalld
+firstsite
+flatpress
+flyspray
+fpm
+fragging
+framesets
+freedns
+friendster
+frontend
+fs
+fsck
+fstab
+galera
+gamemode
+gameplay
+gcc
+gcm
+gem1
+gemfile
+genmask
+gentoolkit
+geoip
+geolocating
+geolocation
+getmail
+getpwent
+gfx
+gfxmode
+girocco
+github
+gitignore
+gitlab
+gitolite
+gitosis
+gitweb
+glibc
+glish
+globals
+globalsieverc
+globalsign
+globbing
+gluster
+gmail
+gmod
+gnulinux
+gnutls
+gogs
+golang
+goto
+gpc
+gpg
+grafana
+graylog
+graylog2
+graylog2
+greenbone
+greylist
+greylisting
+griefing
+groot
+grsecurity
+gsa
+gsad
+guacd
+guarisma
+gui
+gunicorn
+gz
+gzio
+gzip
+gzipped
+gzipping
+h2
+h3
+h4x0r123
+hackage
+hadoop
+halls3
+haproxy
+hardcoded
+hardcopy
+hardlimit
+hashmark
+hasn
+hba
+hd0
+hdfs
+heartbleed
+hiera
+highlite
+hilight
+hilights
```

Comment:
These shouldn't be here: `/applications/messaging/using-irssi-for-internet-relay-chat.md`